dev_tools/docs/run_doctest.py

```diff
@@ -230,17 +229,19 @@ def main():
         'cirq-google/cirq_google/cloud/',
         'cirq-google/cirq_google/api',
```

Comment:
To be on a safe side - let's use replace `api` --> `api/` as for above paths.

Reply:
fixed